Gopkg.lock

```diff
@@ -2,233 +2,184 @@
 
 
 [[projects]]
-  digest = "1:3dcf9df58e425f7ace052d832e683930533be49fcde3b6d66cebb2ed5a97bb3f"
```

Comment:
I think these lines might be getting removed because you have an old version of dep. Can you try updating to the latest and running `dep ensure` again?